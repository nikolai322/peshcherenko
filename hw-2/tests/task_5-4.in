5
1 2 3 4 5
3
-0.01 -0.001 -0.0001
