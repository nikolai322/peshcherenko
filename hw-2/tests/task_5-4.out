15
-0.011100
